libminifi/src/core/Processor.cpp

```diff
@@ -176,7 +176,7 @@ void Processor::removeConnection(std::shared_ptr<Connectable> conn) {
     // Connection is destination to the current processor
     if (_incomingConnections.find(connection) != _incomingConnections.end()) {
       _incomingConnections.erase(connection);
-      connection->setDestination(NULL);
```

Comment:
hmmm, why this change? I would prefer nullptr here. It's easier to read IMO, so I think we should either leave this as-is or use nullptr.

Reply:
I prefer nullptr too, but setDestination now gets a weak_ptr and weak_ptr has no constructor from nullptr_t (or any pointer for that matter).